packages/expo-cli/src/easJson.ts

```diff
@@ -1,19 +1,10 @@
-import { Platform } from '@expo/eas-build-job';
+import { Platform, Workflow } from '@expo/eas-build-job';
 import Joi from '@hapi/joi';
 import fs from 'fs-extra';
 import path from 'path';
 
 // TODO(wkozyra95): move it to @expo/config or to separate package
 
-// Workflow is representing different value than BuildType from @expo/eas-build-job
```

Comment:
is this still true now that it's called [`Workflow` in `@expo/eas-build-job`](https://github.com/expo/turtle-v2/blob/6ffc8aeab05927476e82c5871abdac06896deb5b/src/libs/eas-build-job/src/constants.ts#L1-L4)? if so, we should maybe differentiate between the type names. for the sake of fixing builds right now i assumed they're the same, but we restore this workflow enum later and do whatever needs to be done

Reply:
it's no longer true, originally when we added those types we assumed that client build process will be different from standalone and with dev client, it's almost the same